scripts/ntrip_ros.py

```diff
@@ -196,8 +203,10 @@ def _create_rtcm_msgs_rtcm_message(self, rtcm):
   # Start the node
   rclpy.init()
   node = NTRIPRos()
-  if not node.run():
-    sys.exit(1)
+  # Wait for internet connection
+  is_connected = node.run()
+  if not is_connected:
+    node.wait_for_connection()
```

Comment:
```suggestion
    while not node.run():
        self.get_logger().error(f'Retrying in {NTRIPClient.DEFAULT_RECONNECT_ATEMPT_WAIT_SECONDS} seconds.')    
        time.sleep(NTRIPClient.DEFAULT_RECONNECT_ATEMPT_WAIT_SECONDS)
```

Can't it just be like this without anything else?

Reply:
Yes, it works also like this